specification/validated-relationships-service-api.yaml

```diff
@@ -280,6 +281,81 @@ paths:
                 internalServerError:
                   $ref: '#/components/examples/InternalServerError'
 
+  /Consent:
+    get:
+        summary: Get proxy relationships
+        description: |
+          ## Overview
+          Use this endpoint to get the details about proxy relationships, including current status, based on a provided proxy's NHS Number.
+
+          You can (optionally) include `status={status}` request parameter to filter the limit the response to records with a given status bound to the http://hl7.org/fhir/consent-state-codes CodeSystem e.g. `status=active`.
+
+          You can (optionally) include the `_include=Consent:performer` request parameter to include the proxy's details in the response.
+
+          You can (optionally) include the `_include=Consent:patient` request parameter to include the patient's details in the response.
+
+          ## Sandbox test scenarios
+
+          For details of sandbox test scenarios, or to try out the sandbox using our 'Try it out' feature, see the documentation for each endpoint.
```

Comment:
`see the documentation for each endpoint.` This comment is self referential as this is the endpoint sandbox section for the /Consent endpoint. Maybe the comment could it be removed or changed